### ISSUE TITLE
Doesn't work on http://www.jango.com?

### ISSUE THREAD

**Issue by liar666** [QupZilla/qupzilla#2501]
Hi,

Since qupzilla is not available in the (ubuntu/debian) repositories, I've manually compiled it: 
Application version 2.2.0 (db26f2407f)
QtWebEngine version 5.9.0

It works well, apart on this website: 
http://www.jango.com/stations/263448187/tunein
where the loading wheel spins indefinitely.

I've disabled adblock, enabled all the JS/Cookies stuff, change the useragent, but nothing works...

**Comment by cranes-bill** [comment]
![screenshot from 2017-11-14 21 07 56](https://user-images.githubusercontent.com/5161278/32799287-f075678c-c97f-11e7-9b0f-730ac6bd3f18.png)

**Comment by liar666** [comment]
Would it be possible for you to share your profile?

I've just created a fresh new profile and it still does not work...

Here is what I get (I don't think any of these are related to the problem at hand):
```
$ qupzilla
[24104:24104:1114/204044.092575:FATAL:credentials.cc(155)] Check failed: NamespaceUtils::DenySetgroups(). : Permission denied
#0 0x7f7e7a00945e <unknown>
#1 0x7f7e7a01b16e <unknown>
#2 0x7f7e7a01b419 <unknown>
#3 0x7f7e7acfae74 <unknown>
#4 0x7f7e7acfb4e6 <unknown>
#5 0x7f7e79c7a5d5 <unknown>
#6 0x7f7e79936434 <unknown>
#7 0x7f7e79939b53 <unknown>
#8 0x7f7e797345bf <unknown>
#9 0x7f7e79735755 <unknown>
#10 0x7f7e796d51b1 QtWebEngineCore::BrowserContextAdapter::defaultContext()
#11 0x7f7e76e56c38 QWebEngineProfile::defaultProfile()
#12 0x7f7e7faeef47 MainApplication::MainApplication()
#13 0x55b11147b9aa (/usr/bin/qupzilla+0x55b11147b9a9)
#14 0x7f7e776f81c1 __libc_start_main
#15 0x55b11147bcfa (/usr/bin/qupzilla+0x55b11147bcf9)

QupZilla: 0 extensions loaded
js: Uncaught SecurityError: Blocked a frame with origin "http://staticxx.facebook.com" from accessing a frame with origin "http://www.jango.com". The frame being accessed set "document.domain" to "jango.com", but the frame requesting access did not. Both must set "document.domain" to the same value to allow access.
js: Uncaught SecurityError: Blocked a frame with origin "https://staticxx.facebook.com" from accessing a frame with origin "http://www.jango.com".  The frame requesting access has a protocol of "https", the frame being accessed has a protocol of "http". Protocols must match.

js: Uncaught SecurityError: Blocked a frame with origin "https://staticxx.facebook.com" from accessing a frame with origin "http://www.jango.com".  The frame requesting access has a protocol of "https", the frame being accessed has a protocol of "http". Protocols must match.

js: Uncaught SecurityError: Blocked a frame with origin "https://staticxx.facebook.com" from accessing a frame with origin "http://www.jango.com".  The frame requesting access has a protocol of "https", the frame being accessed has a protocol of "http". Protocols must match.

js: Uncaught SecurityError: Blocked a frame with origin "https://www.facebook.com" from accessing a frame with origin "http://www.jango.com".  The frame requesting access has a protocol of "https", the frame being accessed has a protocol of "http". Protocols must match.

js: Uncaught SecurityError: Blocked a frame with origin "https://staticxx.facebook.com" from accessing a frame with origin "http://www.jango.com".  The frame requesting access has a protocol of "https", the frame being accessed has a protocol of "http". Protocols must match.

js: Uncaught SecurityError: Blocked a frame with origin "https://staticxx.facebook.com" from accessing a frame with origin "http://www.jango.com".  The frame requesting access has a protocol of "https", the frame being accessed has a protocol of "http". Protocols must match.

js: Uncaught SecurityError: Blocked a frame with origin "https://staticxx.facebook.com" from accessing a frame with origin "http://www.jango.com".  The frame requesting access has a protocol of "https", the frame being accessed has a protocol of "http". Protocols must match.

js: Uncaught SecurityError: Blocked a frame with origin "https://www.facebook.com" from accessing a frame with origin "http://www.jango.com".  The frame requesting access has a protocol of "https", the frame being accessed has a protocol of "http". Protocols must match.

```

**Comment by cranes-bill** [comment]
The firs one is a crash, the second is not related. You can try [Appimage](https://www.qupzilla.com/startdownload?appimage), to see how it works.

**Comment by liar666** [comment]
First, it told me that it cannot mix several Qt lib version, so I ran:
unset QT_PLUGIN_PATH
unset QT_QPA_PLATFORM_PLUGIN_PATH
unset LD_LIBRARY_PATH

Then I managed to run it and got the exact same behaviour (spinning wheel that marks loading of the soundtrack never stops spinning).

I removed my .config/qupzilla. Exact same behaviour.

**Comment by cranes-bill** [comment]
Aah, you mean the video can't load. I did understand that the whole site fail to load. The video can't load, because your QtWebEngine version does not support **PROPRIETARY CODECS**.

**Comment by liar666** [comment]
OK Thanks and sorry for not having been clear from the beginning.

To install QupZilla2, I followed the tutorial found here (only one that I found a few months ago):
https://www.linuxhint.com/install-qupzilla-browser-linux/

Unfortunately, using `qt-unified-linux-x64-2.0.3-online.run` does not seems to propose an option to install PROPRIETARY CODECS and also does not allow to modify a previous installation.

Since I don't have time for this at the moment, I'll just forget about QupZilla (and the whole Qt thing) for now.
